src/tests/bulkConcepts/components/Pagination.test.js

```diff
@@ -119,6 +119,28 @@ describe('Test suite for Paginations component', () => {
     wrapper.find('.fa-angle-double-right').simulate('click');
     expect(wrapper.find('Paginations').instance().props.currentPage + 1).toEqual(2);
   });
+  it('should call setPreviousPage ', () => {
```

Comment:
What should call set previous page?

Reply:
I have been more specific with the description

Reply:
I have made adjustments to the description